hack/tools/install_helm.sh

```diff
@@ -5,17 +5,16 @@
 
 source ./common.sh
 
-DESIRED_VERSION=v3.7.0
+HELM_VERSION=v3.8.2
```

Comment:
why it was important to rename the `DESIRED_VERSION`? It is the same env name in all install scripts. Of course, here it is a helm version.

just asking